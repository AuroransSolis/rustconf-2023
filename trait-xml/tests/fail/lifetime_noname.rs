trait_xml::trait_xml! {
    <trait>
        <name>Foo</name>
        <bounds>
            <lifetime>
                <name>'bar</name>
            </lifetime>
            <lifetime>
                <lifetime-bound>'bar</lifetime-bound>
            </lifetime>
        </bounds>
    </trait>
}

fn main() {}
