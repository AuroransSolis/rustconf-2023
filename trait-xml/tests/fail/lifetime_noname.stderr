error: error parsing lifetime parameter: missing lifetime. caller: `::trait_xml::trait_xml_parse_bounds`
  --> tests/fail/lifetime_noname.rs:1:1
   |
1  | / trait_xml::trait_xml! {
2  | |     <trait>
3  | |         <name>Foo</name>
4  | |         <bounds>
...  |
12 | |     </trait>
13 | | }
   | |_^
   |
   = note: this error originates in the macro `$crate::trait_xml_parse_lifetime` which comes from the expansion of the macro `trait_xml::trait_xml` (in Nightly builds, run with -Z macro-backtrace for more info)
